ui/packages/catalog-ui-search/src/main/webapp/component/workspace-sharing/workspace-sharing.view.js

```diff
@@ -143,6 +161,13 @@ define([
         return { value: email }
       })
     },
+    getSharingByAdmin: function() {
+      return (this.model.get('security.access-administrators') || []).map(
+        function(admin) {
```

Comment:
✏️ Could use an arrow function to be more readable. `(admin) => ({value: admin})`